sql-statements/sql-statement-show-placement-for.md

```diff
@@ -11,7 +11,13 @@ summary: The usage of SHOW PLACEMENT FOR in TiDB.
 >
 > If you understand the risks, you can enable this experiment feature by executing `SET GLOBAL tidb_enable_alter_placement = 1;`.
 
-`SHOW PLACEMENT FOR` summarizes all placement options from direct placement and placement policies, and presents them in the canonical form for a specific table, database schema, or partition.
+`SHOW PLACEMENT FOR` summarizes all placement options, and presents them in the canonical form for a specific table, database schema, or partition.
+
+The `Scheduling_State` indicates the current progress the Placement Driver (PD) has made in scheduling the placement:
```

Comment:
```suggestion
The statement returns a table in which the `Scheduling_State` column indicates the current progress that the Placement Driver (PD) has made in scheduling the placement:
```

Reply:
We shouldn't call it a table, it's a result set. Table could be confused with the relational object table.

Reply:
I've modified my comment. PTAL again. thanks~

Reply:
LGTM